### PR TITLE
bump node version

### PR DESCRIPTION
**Pull request by danielpeach**
@anotherchrisberry @zanthrash @icfantv Not sure if there are strong opinions on this, but after #2913, Deck won't start on a node version < 4.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "zone.js": "^0.6.23"
   },
   "engines": {
-    "node": ">=0.10.0",
+    "node": ">=4.0.0",
```
Comment (anotherchrisberry) [inline package.json]: While you're at it, can you add ` <6.0` to indicate it won't run on node 6? (not sure what the actual syntax is, the `.0` might be unneeded)

**Comment by icfantv** [comment]
I don't have a strong opinion.

**Comment by danielpeach** [comment]
Is this OK to merge (after a rebase)?

**Comment by anotherchrisberry** [comment]
@danielpeach yessir

**Pull request merged**